scarlet/source.py

```diff
@@ -358,7 +361,7 @@ def update(self):
 
 class ExtendedSource(PointSource):
     def __init__(self, frame, sky_coord, observation, bg_rms, thresh=1,
-                 symmetric=False, monotonic=True, center_step=5, delay_thresh=10, **component_kwargs):
+                 symmetric=True, monotonic=True, center_step=5, delay_thresh=10, **component_kwargs):
```

Comment:
Why should point sources be made symmetric?

Reply:
Testing shows that symmetry does improve deblending and this was the default scarlet setting before the API change, it was just turned off because it wasn't working properly. Now that it is I think we should return to the expected default.

Reply:
I surrender to the impirical argument, but isn't it a little strange? Aren't point source supposed to be intrinsically symmetric?

Reply:
This is actually an `ExtendedSource`, which inherits from `PointSource`. And `PointSource` is actually a bit of a misnomer in that it is a source that is initialized as a point source but then is allowed to have its morphology updated. It is only a true point source if `fix_morph=True` (which might be a good idea to make the default behavior so that it really is a point source).

Reply:
Yup! Completely misread that! Sorry.
I don't have anything against living a bit of freedom to point sources morphologies unless we know for sure something is a point source (star).

Reply:
It's actually even more complicated because even point sources should have their positions updated (since it is highly unlikely that they will be exactly centered). In fact, this should probably be an issue.